package.json

```diff
@@ -83,7 +83,7 @@
     "@okta/ci-pkginfo": "1.2.0",
     "@okta/ci-update-package": "1.4.0",
     "@okta/courage": "1.3.0-beta.30",
-    "@okta/i18n": "19.17.0",
+    "@okta/i18n": "19.21.0",
```

Comment:
Man, I can't wait until we use bundledDependencies for this, so we don't have to include node_modules in our repo.